civictechprojects/admin.py

```diff
@@ -9,6 +9,10 @@ class ProjectAdmin(admin.ModelAdmin):
     list_display =  tuple(project_text_fields) + ('project_creator',) + project_filter_fields
     search_fields = project_text_fields + ['project_creator__email']
     list_filter = project_filter_fields
+    def save_model(self, request, obj, form, change):
+        if change:
+            obj.recache(recache_linked=True)
+        super().save_model(request, obj, form, change)
```

Comment:
This recaching seems to be working even though the recache is done before the save operation, but let's put the operations in the save -> recache order for consistency if nothing else.

Reply:
Not sure how I missed that one. Fixing.